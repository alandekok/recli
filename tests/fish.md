# one

Can you count this high?

## one fish

Swims in ocean blue.

# two

Comes after one.

## two fish

Swim beside each other.

# red

Color of sunset.

## red fish

Swims beneath the waves.

# blue

Sunny sky is bright.

## blue fish

Reflecting the sky.
