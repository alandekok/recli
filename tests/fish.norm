(blue|one|red|two) fish
