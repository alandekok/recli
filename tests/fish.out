two fish
^ - No permission
blue fish
^ - No permission
two fish
^ - No permission
